modules/auth/auth-core/src/main/scala/auth/core/module/SilhouetteModule.scala

```diff
@@ -25,7 +25,7 @@ sealed class SilhouetteModule extends AbstractModule with ScalaModule {
     bind[IDGenerator].toProvider[IDGeneratorProvider]
 
     bind[ExecutionContext]
-      .annotatedWith[SilhouetteContext]
+      .annotatedWith[SilhouetteContext] // TODO what does this do?
```

Comment:
This is how I tell it - any EC that is annotated by SilhouetteContext should be provided by  that belowmentioned provider.